packages/realtime-api/src/event-handler.ts

```
@@ -0,0 +1,126 @@
+    for (const handler of handlers) {
+      handler(eventName, event);
+    }
+
+    const allHandlers = (
+      this.eventHandlers[EventNames.ALL] || []
+    ).slice() as Function[];
+    for (const handler of allHandlers) {
+      handler(eventName, event);
+    }
+
+    const allClientHandlers = (
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)